db/schema.rb

```diff
@@ -10,9 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2021_11_09_150717) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "monsters", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "habitat"
+    t.text "description", null: false
+    t.string "image", null: false
+    t.string "classification"
+    t.integer "votes", default: 0
```

Comment:
Voting would be its own feature, separate from viewing a list of Monsters, and not something we recommend tackling until other core functionality is added! Take a look at the suggested timeline for prioritizing features

_Votes should have their own table rather than the data living with the resource, you can come back to this feature later!_